src/main/java/htsjdk/samtools/cram/encoding/reader/MultiRefSliceAlignmentSpanReader.java

```diff
@@ -63,28 +62,46 @@ public MultiRefSliceAlignmentSpanReader(final BitInputStream coreInputStream,
         this.currentAlignmentStart = initialAlignmentStart;
 
         for (int i = 0; i < recordCount; i++) {
-            readCramRecord();
+            final CramCompressionRecord cramRecord = new CramCompressionRecord();
+            super.read(cramRecord);
+            processRecordSpan(cramRecord);
         }
     }
 
     public Map<Integer, AlignmentSpan> getReferenceSpans() {
         return Collections.unmodifiableMap(spans);
     }
 
-    private void readCramRecord() {
-        final CramCompressionRecord cramRecord = new CramCompressionRecord();
-        super.read(cramRecord);
-
+    private void processRecordSpan(final CramCompressionRecord cramRecord) {
```

Comment:
this did two jobs (reading and processing) and was only named after the first.  I moved reading outside this function and renamed it to processing.